docs/source/usage.rst

```diff
@@ -16,6 +19,48 @@ In the template (e.g. post_detail.) add the following template tags where ``obj`
     {% render_comments obj request %}  {# Render all the comments belong to the passed object "obj" #}
 
 
+2. Displaying Comment Count in Post Page:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In ``views.py`` under the post view add the following line at the top of your page:
+
+.. code:: python
+
+    from comment.models import Comment
+    
+    ...
+    def post_detail(request, pk):
```

Comment:
Thanks for contributing!
What the purpose of passing the comment's query to post detail template? The comments are already available on a particular model object via `render_comments` function!

Reply:
Sorry for late reply. 
I think I did this because from what I could tell the render_comments function worked on the details page but not on the index / list page where I also wanted to show the number of comments to a post without rendering them.

If, I'm wrong please correct me.
![image](https://user-images.githubusercontent.com/5003973/96304382-e944c580-0ff3-11eb-98c4-b38b85c7c127.png)

Reply:
> how do i can change the default ordering comments for the products or content types in my project????
> i want the comments be ordered by oldest to the newest

This will do the trick if you render the comment from your view
```python
Comment.objects.filter_parents_by_object(Post.objects.all().first()).order_by('posted')
```

However, I don't think this is possible when rendering the comments from the template using `render_comment`. 
You might open a feature request to allow custom ordering.